.github/sync-repo-settings.yaml

```diff
@@ -17,6 +17,7 @@ branchProtectionRules:
       - units (8)
       - units (11)
       - 'Kokoro - Test: Integration'
+      - 'Kokoro - Against Pub/Sub Lite samples'
```

Comment:
you're going to want to add this and the build.sh file to the synth.py script to exclude it. Otherwise, next time synthtool runs it will try and revert your handwritten changes.

Reply:
Done.